csd/scripts/control.sh

```diff
@@ -41,15 +41,32 @@ move_aux_files() {
     fi
 }
 
-edit_variables() {
+get_property() {
+    local file=$1
+    local key=$2
+    local line=$(grep "$key=" ${file}.properties | tail -1)
+    echo "${line/$key=/}"
+}
+
+load_variables() {
     HWX_REGISTRY_SP_DB_PORT=":${HWX_REGISTRY_SP_DB_PORT}"
     if [ "${HWX_REGISTRY_SP_DB_PORT}" == ":0" ]; then
         HWX_REGISTRY_SP_DB_PORT=""
     fi
+
+    # Logging Variables
+    export HWX_REGISTRY_LOG_THRESHOLD=$(get_property log4j log.threshold)
```

Comment:
If I used logback, we would be parsing an XML file from bash, doable... but more complicated.